src/main/scala/com/blackmorse/spark/clickhouse/utils/SqlUtils.scala

```diff
@@ -0,0 +1,19 @@
+package com.blackmorse.spark.clickhouse.utils
+
+import com.blackmorse.spark.clickhouse.tables.ClickhouseTable
+
+object SqlUtils {
+
+  def prepareSqlQuery(
+                       fields: String,
+                       table: ClickhouseTable,
+                       useForceCollapsingModifier: Boolean
+                     ): String = {
+    val sql = s"SELECT $fields FROM $table"
+    table.engine match {
+      case engine if useForceCollapsingModifier &&
```

Comment:
I see no reason for using patter-matching. Probably just if-else will work

Reply:
Done